### ISSUE TITLE
bdhcalc-1.1 failed to build on x86_64

### ISSUE THREAD

**Issue by nysnatuss** [haikuports/haikuports#1982]
[haiku-apps/bdhcalc-1.1](https://github.com/haikuports/haikuports/blob/master/haiku-apps/bdhcalc/bdhcalc-1.1.recipe) failed to build on x86_64.

Error: ([Full log](https://pastebin.com/DgyFtWSV))
```
In file included from src/bdh_main.cpp:6:0:
src/bdh_view.h:38:21: error: extra qualification 'bdh_view::' on member 'pwr2' [-fpermissive]
  unsigned long long bdh_view::pwr2(unsigned long long power);
                     ^
src/bdh_view.h:39:21: error: extra qualification 'bdh_view::' on member 'pwr10' [-fpermissive]
  unsigned long long bdh_view::pwr10(unsigned long long power);
                     ^
In file included from /boot/system/develop/headers/os/interface/Alert.h:11:0,
                 from /boot/system/develop/headers/os/InterfaceKit.h:6,
                 from src/bdh_main.cpp:1:
/boot/system/develop/headers/os/interface/Window.h: In member function 'virtual void bdh_window::MessageReceived(BMessage*)':
/boot/system/develop/headers/os/interface/Window.h:291:18: error: 'typedef class BLooper BWindow::inherited' is private
  typedef BLooper inherited;
                  ^
src/bdh_main.cpp:44:2: error: within this context
  inherited::MessageReceived(msg);
  ^
```
I think we can just remove the `bdh_view::` and make `typedef class BLooper BWindow::inherited` public.

**Comment by Oyami-Srk** [comment]
Fixed.

**Comment by nysnatuss** [comment]
Fixed in #1993.
